_posts/en/newsletters/2020-08-26-newsletter.md

```diff
@@ -0,0 +1,109 @@
+---
+title: 'Bitcoin Optech Newsletter #112'
+permalink: /en/newsletters/2020/08/26/
+name: 2020-08-26-newsletter
+slug: 2020-08-26-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter links to a discussion about routed coinswaps and
+includes our regular sections with summaries of questions and answers
+from the Bitcoin StackExchange, releases and release candidates, and
+notable changes to popular Bitcoin infrastructure software
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Discussion about routed coinswaps:** Chris Belcher [posted][belcher
+  coinswap] a design document for a routed multi-transaction coinswap
+  implementation to the Bitcoin-Dev mailing list as a follow-up to his
+  previous post in May (see [Newsletter #100][news100 coinswap]).
+  Discussion from ZmnSCPxj and Nadav Kohen focused on ensuring the
+  protocol was safe both in its use of cryptography and in ensuring that
```

Comment:
Remove repeated ensuring: "ensuring the protocol was safe in ... ensuring that ..."